setup/symfony_cli.rst

```diff
@@ -580,6 +580,8 @@ The ``.symfony.local.yaml`` file provides advanced configuration options:
         p12: path/to/custom-cert.p12
         # toggle GZIP compression
         use_gzip: true
+        # enable Cross-origin resource sharing (CORS) for all request
+        allow_cors: true
```

Comment:
This is the source code where this option is defined:

https://github.com/symfony-cli/symfony-cli/blob/132611d2ea65a07a449b3e86ab4022f1ac5acb8f/local/project/config.go#L259